layers/+misc/nlinum/funcs.el

```diff
@@ -0,0 +1,20 @@
+;;; packages.el --- nlinum Layer packages File
+;;
+;; Copyright (c) 2012-2016 Sylvain Benner & Contributors
+;;
+;; Author: Thomas de Beauchêne <thomas.de.beauchene@gmail.com>
+;; URL: https://github.com/syl20bnr/spacemacs
+;;
+;; This file is not part of GNU Emacs.
+;;
+;;; License: GPLv3
+
+(defun spacemacs/nlinum-on ()
```

Comment:
Why not, also makes sense.
It was inspired from the `linum-on` function, used in `define-globalized-minor-mode` to define `global-linum-mode`, which purpose is similar.
But yeah, `maybe` might be more explicit.